src/studio/src/designer/frontend/shared/version-control/versionControlHeader.test.tsx

```diff
@@ -26,15 +26,20 @@ const handlers = [
     },
   ),
 ];
-const server = setupServer(...handlers);
+const versionControlHeaderMockServer = setupServer(...handlers);
 
-beforeAll(() => {
+export const versionControlHeaderBeforeAll = () => {
   (window as Window as IAltinnWindow).org = 'test-org';
   (window as Window as IAltinnWindow).app = 'test-app';
-  server.listen();
-});
-afterEach(() => server.resetHandlers());
-afterAll(() => server.close());
+  versionControlHeaderMockServer.listen();
+};
+export const versionControlHeaderafterEach = () =>
+  versionControlHeaderMockServer.resetHandlers();
+export const versionControlHeaderafterAll = () =>
+  versionControlHeaderMockServer.resetHandlers();
+beforeAll(versionControlHeaderBeforeAll);
+afterEach(versionControlHeaderafterEach);
+afterAll(versionControlHeaderafterAll);
```

Comment:
I don't see how this helps anything?
More lines and frequent of words that are implied from the filename.
Also there is no need to export these as far as I can see?

Reply:
These are reused in another test, so that is why they are exported. Its to solve the fact that this class component is doing some rest-calls inside the compontent. This branch doesn't aim to solve all problems, so this seemed to be the best solution.